src/dsp_tools/commands/excel2json/models/new_lists_deserialise.py

```diff
@@ -13,5 +13,6 @@ class ExcelFile:
 
 @dataclass
 class ExcelSheet:
+    excel_name: str
```

Comment:
it's a bit subpar that in the class `ExcelFile`, the same information is named `filename`. I would prefer `filename` for both classes

Reply:
because the class ExcelFile will be removed in the end. So that difference will not be there in the end. And excel name feels more explicit to me.